### DISCUSSION TITLE
requirements.bat

### DISCUSSION THREAD

**Discussion by gaydarr** [winstxnhdw/lc-hax#474] [Q&A]
repository cloned utilizing git bash ✅
executed requirements.bat ✅
error: 
![image](https://github.com/winstxnhdw/lc-hax/assets/163698054/710a0698-ced2-4b5f-a390-4cbfa3f1d2b0)

git is installed
later version of net 8.0 installed
any solutions?

**Comment by winstxnhdw** [comment]
antivirus issue

**Reply by gaydarr** [reply]
If windows defender was the reason I would receive a specific warning about it in command prompt like this

![image](https://github.com/winstxnhdw/lc-hax/assets/163698054/e65d9dd2-276d-4bcd-b6de-0441606fa5fb)

Do you have reason to believe that's the reason anyways?  I did in fact disable it for the time being and still having the same issues.  If you see no other reasons I'll just close the discussion.

**Reply by winstxnhdw** [reply]
Is there anything special about your computer you aren’t telling?

**Reply by winstxnhdw** [reply]
Anyways, instead of disabling your antivirus, I recommend you to whitelist it as written in the README. It’s safer than nuking defender completely.

**Reply by winstxnhdw** [reply]
If all else fails, you can always use another injector of your choice.
